src/lib/certificate-manager.ts

```diff
@@ -148,11 +152,51 @@ export async function openCertificate() {
 	shell.showItemInFolder( CA_CERT_PATH );
 }
 
+/**
+ * Checks if the root CA certificate is already trusted by the system
+ * @returns A promise that resolves to true if the certificate is trusted, false otherwise
+ */
+export async function isRootCATrusted(): Promise< boolean > {
+	if ( ! fs.existsSync( CA_CERT_PATH ) ) {
+		return false;
+	}
+
+	const platform = process.platform;
+	if ( platform === 'win32' ) {
```

Comment:
```suggestion
	if ( process.platform === 'win32' ) {
```
Nit, but I don't see why we assign this value to a variable.

Reply:
I see that you like having the minimum number of lines of code :)